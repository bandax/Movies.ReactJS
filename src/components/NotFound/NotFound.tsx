import * as React from 'react';

const NotFound: React.FunctionComponent = () => (
  <div className="app">Page Not Found</div>
);

export { NotFound };
