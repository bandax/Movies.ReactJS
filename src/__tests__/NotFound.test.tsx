import * as React from 'react';
import { NotFound } from '../components/NotFound/NotFound';
import { render, fireEvent, waitFor, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import '@testing-library/jest-dom/extend-expect';

describe('<NotFound />', () => {
  it('render a Header Component', async () => {
    const { asFragment } = render(<NotFound />);
    expect(asFragment()).toMatchSnapshot();
  });
});
