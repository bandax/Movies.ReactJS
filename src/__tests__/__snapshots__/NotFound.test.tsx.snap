// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> render a Header Component 1`] = `
<DocumentFragment>
  <div
    class="app"
  >
    Page Not Found
  </div>
</DocumentFragment>
`;
